CryptoPrices/Sources/Home/Tests/HomeTests/HomeViewModelSpec.swift

```diff
@@ -9,16 +16,24 @@ import UseCaseProtocol
 final class HomeViewModelSpec: QuickSpec {
 
     override func spec() {
+
         var homeViewModel: HomeViewModel!
         var myCoinsUseCase: MockMyCoinsUseCaseProtocol!
+        var trendingCoinsUseCase: MockTrendingCoinsUseCaseProtocol!
 
         describe("the HomeViewModel") {
+
             beforeEach {
                 myCoinsUseCase = MockMyCoinsUseCaseProtocol()
-                homeViewModel = await HomeViewModel(myCoinsUseCase: myCoinsUseCase)
+                trendingCoinsUseCase = MockTrendingCoinsUseCaseProtocol()
+                homeViewModel = await HomeViewModel(
+                    myCoinsUseCase: myCoinsUseCase,
+                    trendingCoinsUseCase: trendingCoinsUseCase
+                )
             }
 
             describe("its initial state") {
+
                 it("has the correct value for myCoins") {
                     await expect {
                         await homeViewModel.myCoins
```

Comment:
Please add a test for `it("has the correct value for trendingCoins")` in `"its initial state"`